pkg/controller/provider/controller.go

```diff
@@ -231,6 +231,9 @@ func (r Reconciler) Reconcile(request reconcile.Request) (result reconcile.Resul
 			"Waiting connection tested or inventory created.")
 		result.RequeueAfter = base.SlowReQ
 	}
+	if provider.Status.HasCondition(ConnectionAuthFailed) {
+		result.RequeueAfter = 0
```

Comment:
does it change when we post an update to the provider?

Reply:
yes, when we change some things on which we have watch, the Reconcile gets triggered again.
https://github.com/konveyor/forklift-controller/blob/76ee1608b4850a0d47e92b508627b4e94847e29c/pkg/controller/provider/controller.go#L101-L114

Reply:
ok, makes sense